lms/static/scripts/frontend_apps/components/dashboard/OrderableActivityTable.tsx

```diff
@@ -46,7 +46,7 @@ export default function OrderableActivityTable<T>({
       columns.map(({ field, label }, index) => ({
         field,
         label,
-        classes: index === 0 ? 'w-[60%]' : undefined,
+        classes: index === 0 ? 'lg:w-[60%] md:w-[45%]' : undefined,
```

Comment:
If we reserve 60% of space for the first column, there's not enough space for the rest in medium resolutions, so this reduces the reserved space to 45% in medium resolutions, and not reserved at all for small resolutions.